src/components/authpage/LoginPage.jsx

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from "react";
+import { NavLink } from "react-router-dom";
+import { useAuth } from "../../context/AuthProvider";
+// import { useDocumentTitle } from "../../utils";
+import "./authpage.css"
+
+const guestCredentialsStyle = {
+    backgroundColor: "transparent",
+    border: 0,
+    cursor:"pointer"
+}
+
+const emailMatchPattern =
+    /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+export default function LoginPage() {
+    const [email, setEmail] = useState("");
+    const [emailError, setEmailError] = useState(false);
+    const [passwordError, setPasswordError] = useState(false);
+    const [password, setPassword] = useState("");
+    const [showPassword, setShowPassword] = useState(false);
```

Comment:
You can use 1 useState and add all these fields in an object maybe? Do let me what you think.